xbmc/cores/VideoPlayer/DVDCodecs/Video/DVDVideoCodecAndroidMediaCodec.cpp

```diff
@@ -788,7 +788,9 @@ void CDVDVideoCodecAndroidMediaCodec::Dispose()
   if (m_codec)
   {
     m_codec->stop();
+    xbmc_jnienv()->ExceptionClear();
```

Comment:
Btw. You get an IllegalStateException right? Why not checking if state of mediacodec is initialized? Then no need to clear the exception?

Reply:
All fine, I did not consider errors, that throw this exception and is not forwarded to us.